recirq/quantum_chess/experiments/circuit_transform_benchmark.py

```diff
@@ -0,0 +1,129 @@
+"""
+Benchmarking tool for logical-to-hardware circuit transformations.
+
+To run this:
+  python recirq/quantum_chess/experiments/circuit_transform_benchmark.py \
+      file1.qasm [file2.qasm [...]]
+
+This script takes 1 or more positional arguments which are the paths to qasm
+circuit files.
+
+For each file, this script will:
+  1) load the file into a cirq Circuit
+  2) optimize/decompose the circuit for the Sycamore chip
+  3) apply the CircuitTransformer
+  4) validate the resulting circuit for the Sycamore chip
+
+Some statistics about the transformation in step (3) will be printed, including
+timing and increases to the circuit size.
+
+Any qasm format circuit file should work.
+See https://github.com/BensonZhou1991/circuittransform for some examples which
+are commonly used in publications.
+"""
+
+import cirq
+import cProfile
+import pstats
+import sys
+
+import recirq.quantum_chess.circuit_transformer as ct
+from cirq.contrib.qasm_import import circuit_from_qasm
+from typing import List
+from timeit import default_timer as timer
+
+
+def print_stats(time_sec: float, circuit: cirq.Circuit) -> None:
```

Comment:
Considering adding a short docstring to this function and the functions below.

Reply:
Updated the docstrings in this file.